www/js/recent.js

```diff
@@ -1,313 +0,0 @@
-angular.module('emission.main.recent', ['ngCordova', 'emission.services'])
```

Comment:
Again, I think that this should be retained and we should just navigate to the states from the HTML. It's bad to blow up main.js when it is not necessary. Lots of small files are better than one humongous file.